expression/builtin_math.go

```diff
@@ -190,8 +192,85 @@ func builtinRound(args []types.Datum, ctx context.Context) (d types.Datum, err e
 
 // See http://dev.mysql.com/doc/refman/5.7/en/mathematical-functions.html#function_conv
 func builtinConv(args []types.Datum, ctx context.Context) (d types.Datum, err error) {
-	//TODO  implement
-	return d, errors.New("Function unimplement")
+	var (
+		n        string
+		toBase   int64
+		fromBase int64
+		signed   bool
+		negative bool
+		touval   bool
```

Comment:
it means when tobase < 0 , we let n to unsigned ignore the sign.  example  -15 to 15,  15 to 15. if tobase > 0, n will be forced conversion.